### ISSUE TITLE
Install script doesn't preserves package versions for yq and jq

### ISSUE THREAD

**Issue by bitphage** [stefanprodan/kube-tools#49]
On recent kube-tools upgrade our github pipeline break due to upgrade of `yq` version from 3 to 4 with this change: https://github.com/stefanprodan/kube-tools/commit/642a102a4c3ddef221a4c59be125212a7e884139

We were using branch v1 in the pipiline step: `uses: stefanprodan/kube-tools@v1`

And now we cannot just switch to a tag to achieve proper version pin (`uses: stefanprodan/kube-tools@v1.5.0`) because install script doesn't have a version pin. https://github.com/stefanprodan/kube-tools/blob/v1.5.0/src/install.sh
Basically it means that docker image rebuild is not giving reproducible result, and that's really bad.
